src/openrct2-ui/windows/GuestList.cpp

```diff
@@ -381,6 +381,10 @@ class GuestListWindow final : public Window
 
     void OnDropdown(WidgetIndex widgetIndex, int32_t dropdownIndex) override
     {
+        if (dropdownIndex == -1)
```

Comment:
I've added protetion for -1 dropdownIndex for all windows this PR just adds it to the small selection of windows that were missing it.

Reply:
Perhaps i worded this wrong. There are a selection of windows that when it receives -1 it enters a default behaviour mode. The rest of the windows should just return on -1. I've reviewed every single dd event and ensured that the ones that don't have a default event have the return so in the process I've added a few missing returns. I can't see any way you could remove the returns unless you split the dd event into two but that's a bit messy.